resources/services/mq/brokers.go

```diff
@@ -274,31 +265,53 @@ func MqBrokers() *schema.Table {
 						Type:        schema.TypeJSON,
 					},
 				},
+				Relations: []*schema.Table{
+					{
+						Name:          "aws_mq_broker_configuration_revisions",
+						Resolver:      fetchMqBrokerConfigurationRevisions,
+						IgnoreInTests: true,
```

Comment:
can we add terraforms to test this? (we can do it in a separate PR, merge it and then tests will pass here and we can merge this)

Reply:
objects are already there. no need in terraform